backend/api/v1/project_service.go

```diff
@@ -426,7 +428,119 @@ func (s *ProjectService) SetIamPolicy(ctx context.Context, request *v1pb.SetIamP
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
-	return convertToV1IamPolicy(ctx, s.store, iamPolicyMessage)
+
+	v1IamPolicy, err := convertToV1IamPolicy(ctx, s.store, iamPolicyMessage)
+	if err != nil {
+		return nil, err
+	}
+	v1IamPolicy.BindingDeltas = findIamPolicyDeltas(oriIamPolicy, iamPolicyMessage.Policy)
+	return v1IamPolicy, nil
+}
+
+type bindMapKey struct {
+	User string
+	Role string
+}
+
+type condMap map[exportedExpr]bool
+
+// only contains exported fields in expr.Expr.
+type exportedExpr struct {
+	Expression  string
+	Title       string
+	Description string
+	Location    string
+}
+
+func convertToExportedExpr(e *expr.Expr) *exportedExpr {
+	if e == nil {
+		return nil
+	}
+	return &exportedExpr{
+		Expression:  e.Expression,
+		Title:       e.Title,
+		Description: e.Description,
+		Location:    e.Location,
+	}
+}
+
+func convertToOriExpr(e *exportedExpr) *expr.Expr {
+	if e == nil {
+		return nil
+	}
+	return &expr.Expr{
+		Expression:  e.Expression,
+		Title:       e.Title,
+		Description: e.Description,
+		Location:    e.Location,
+	}
+}
+
+func findIamPolicyDeltas(oriIamPolicy *storepb.IamPolicy, newIamPolicy *storepb.IamPolicy) []*v1pb.BindingDeltas {
+	deltas := []*v1pb.BindingDeltas{}
```

Comment:
```suggestion
	var deltas []*v1pb.BindingDeltas
```

Reply:
For consistency with the rest of the code. Use var declarative whenever possible.